src/components/Proposal/SentimentSurvey/SentimentSurveyRow.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useCallback, useEffect, useState } from 'react'
+
+import useFormatMessage from 'decentraland-gatsby/dist/hooks/useFormatMessage'
+import TokenList from 'decentraland-gatsby/dist/utils/dom/TokenList'
+
+import { ReactionType, Topic } from '../../../entities/SurveyTopic/types'
+import { REACTIONS_VIEW } from '../../../entities/SurveyTopic/utils'
+import IconHelper from '../../Helper/IconHelper'
+import AddReaction from '../../Icon/AddReaction'
+import Cross from '../../Icon/Cross'
+
+import './SentimentSurveyRow.css'
+
+interface Props {
+  topic: Topic
+  reaction?: ReactionType | null
+  onReactionPicked: (topic: Topic, reaction: ReactionType) => void
+  onReactionUnpicked: (topic: Topic) => void
+}
+
+const SentimentSurveyRow = ({ topic, reaction, onReactionPicked, onReactionUnpicked }: Props) => {
+  const t = useFormatMessage()
+  const [showAddReaction, setShowAddReaction] = useState(false)
+  const [showReactions, setShowReactions] = useState(false)
+  const [pickedReaction, setPickedReaction] = useState<ReactionType | null>()
+  const reactionPicked = pickedReaction != null
+
+  const pickReaction = useCallback(
+    (reaction: ReactionType) => {
+      setShowReactions(false)
+      setPickedReaction(reaction)
+      onReactionPicked(topic, reaction)
+    },
+    [onReactionPicked, topic]
+  )
+
+  const changeReaction = useCallback(() => {
+    setPickedReaction(null)
+    onReactionUnpicked(topic)
+    setShowReactions(true)
+    setShowAddReaction(true)
+  }, [onReactionUnpicked, topic])
+
+  useEffect(() => {
+    if (reaction && reaction !== ReactionType.EMPTY) {
+      setShowReactions(false)
+      setPickedReaction(reaction)
+    }
+  }, [reaction])
+
+  return (
+    <div
+      className={TokenList.join(['SentimentSurveyRow', showAddReaction && 'SentimentSurveyRow__Expanded'])}
+      onMouseEnter={() => !reactionPicked && setShowAddReaction(true)}
```

Comment:
You can avoid this logic using css and the :hover pseudo-class in SentimentSurveyRow and changing the opacity in SentimentSurveyRow__AddReactionLabel.